### PR TITLE
Added sort by fiat balance to endpoint

### LINKED ISSUE #318: Sort assets by fiat value
**Is your feature request related to a problem? Please describe.**
- On safe-react we will disable being able to manually sort the asset list with the next release, purely relying on the client-gateway sorting.
- We currently sort the list alphabetically
- To evaluate whether this is the best approach, I ran a quick poll on Twitter, showing a rather clear picture
image

**Describe the solution you'd like**
- Sort asset list in client-gateway by fiat value to show bigger portfolio positions at the top.

**Describe alternatives you've considered**
- We might at some point re-introduce manual sorting in some form: https://github.com/gnosis/safe/issues/381 but the default sorting should stick with what would be the best solution for most users. (which according to the poll above seems to be sorting by fiat value)

### PR DESCRIPTION
**Pull request by jpalvarezl**
Closes #318 

https://stackoverflow.com/a/60916195/7721253 => Benchmark of possible ways to solve this

Sorting by fiat balance for the `balances` endpoint.

### CODE REVIEW

File: src/services/balances.rs [hunk 1]
```diff
@@ -45,6 +46,14 @@ pub fn balances(
         })
         .collect();
 
+    service_balances.sort_by(|a, b| {
+        a.fiat_balance
+            .parse::<f64>()
```
Comment (rmeissner) [inline src/services/balances.rs:51]: Any issues here, that value might be to large or have too many decimals?

Comment (jpalvarezl) [reply src/services/balances.rs:51]: I would have to add unit tests for this parse method. As we use it in this same file too in line 44.

Comment (rmeissner) [reply src/services/balances.rs:51]: I see we fallback to 0 ... so doesn't matter for now :)

Comment (jpalvarezl) [reply src/services/balances.rs:51]: Not sure what the alternative would be however.

File: src/services/balances.rs [hunk 2]
```diff
@@ -45,6 +46,14 @@ pub fn balances(
...
+            .unwrap_or(0.0)
+            .partial_cmp(&b.fiat_balance.parse::<f64>().unwrap_or(0.0))
+            .unwrap_or(Ordering::Equal)
+    });
+    service_balances.reverse();
```
Comment (rmeissner) [inline src/services/balances.rs:56]: Wouldn't it be easier to just swap `a` and `b` in the `sort_by`?

File: src/services/balances.rs (hunk 1 above)
Comment (jpalvarezl) [reply src/services/balances.rs:51]: Should we introduce something like this: https://docs.rs/rug/0.6.0/rug/ ?

File: src/services/balances.rs (hunk 2 above)
Comment (jpalvarezl) [reply src/services/balances.rs:56]: absolutely, well spotted.

**Review by rmeissner — APPROVED** [review]

**Pull request merged**